### ISSUE TITLE
Still active

### ISSUE THREAD

**Issue by Hermanverschooten** [andrewtimberlake/elixir-pdf#1]
Hi,

Are you still maintaining this repo and it's hex package?
I noticed the hex package contains completely different code.

I fixed a number of bugs with regards to the `xref` table.
If you like I can make a PR of these.

**Comment by Hermanverschooten** [comment]
I created 2 PR's
 - #2 includes some fixes
 - #3 allows adding an AFM-font

**Comment by andrewtimberlake** [comment]
Hi Herman
I originally created the package as a quick wrapper around the erlguten pdf code to "elixify" it for better piping. I have that code running in production.
I then began the mammoth task of starting a PDF library from scratch which is the current code.
It is active in the sense that I still intend to move forward with it but to date have not had time to work on it. I have a production product that needs PDF capability which is what it was intended for.
Help is definitely welcome!

**Comment by Hermanverschooten** [comment]
Hey Andrew,

I too have the need of an Elixir PDF lib.  
I initially dreamt of porting prawn pdf from ruby, but ...
I found your code to be very good, but ran into some issues, especially with the size calculations and the locations written to the `xref` section.
I mostly solved them by using `to_iolist` and counting the real bytes.
I am currently working on adding basic graphics; stroke/non-stroke color, line widths, line and rectangle are already implemented.
I'd love for you to replace the version on hex with at least what's in the repo now.

I am going to keep working on this for some time more, firstly to implement all the features I need for porting several rails apps to elixir, the lack of good PDF support was what kept me back.

If you can find the time to look over the PR's, I would be most obliged.

**Comment by andrewtimberlake** [comment]
Thanks Herman
I have merged your current changes.
I was hoping to avoid running `to_iolist` and then calculating size but not sure if that was premature optimisation.
I also started this because of a need to port Prawn code to elixir—currently I’m still running Ruby processes to generate some PDF because we have other, more pressing production issues that always get in the way ;-)
I’ll release this as `0.1.0` and we can move from there.
